internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -144,9 +147,25 @@ func (e *EnvStackConfig) Parameters() ([]*cloudformation.Parameter, error) {
 			ParameterValue: aws.String(fmt.Sprintf(fmtServiceDiscoveryEndpoint, e.in.Name, e.in.App.Name)),
 		},
 		{
-			ParameterKey:   aws.String(createHTTPSListener),
+			ParameterKey:   aws.String(envParamCreateHTTPSListenerKey),
 			ParameterValue: aws.String(httpsListener),
 		},
+		{
```

Comment:
Was there a reason why you added these boss? there is a default in the template that's `""`

Reply:
discussed offline. It is because otherwise `env upgrade` would think we are trying to deprecate a parameter when we upgrade the env. I'll send a follow-up PR to include an integ test, making sure the `Parameter()` sets all the environment parameters we have in the template.